common_features.mk

```diff
@@ -291,6 +293,8 @@ endif
 
 include $(DRIVER_PATH)/qwiic/qwiic.mk
 
+QUANTUM_LIB_SRC += $(LIB_PATH)/lib8tion/lib8tion.c
```

Comment:
@pelrun Not sure where this line belongs in this file.

Reply:
Probably best to stick it next to the RGB_MATRIX_ENABLE block, since that's pretty much what uses it.

Reply:
@pelrun Thanks!